helix-view/src/env.rs

```diff
@@ -0,0 +1,37 @@
+use std::{fmt::Display, path::PathBuf};
+
+use crate::Document;
+pub struct Env {
```

Comment:
This type can be eliminated: replacing the environment in a string can be done as a single function rather than a type+method.

Reply:
I've converted to this style now. I opted for `inject_into_with(iterator, doc)` another way to go could be `inject_with_into(doc, iterator)` which may be better?

Reply:
I prefer `(iterator, doc)`